src/main/resources/templates/visa/dashboard.html

```
@@ -0,0 +1,181 @@
+            <tbody>
+            <tr th:each="visa : ${visas}">
+                <td th:text="'# ' + ${visa.id}">#101</td>
+
+                <td th:if="${currentUser.userAuthorization.name() != 'USER'}"
+                    th:text="${visa.applicantName}">Anna Andersson</td>
+
+                <td th:if="${currentUser.userAuthorization.name() != 'USER'}"
+                    th:text="${visa.handlerName != null ? visa.handlerName : 'Unassigned'}">-</td>
+
+                <td th:text="${visa.visaType}">STUDENT</td>
+                <td>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </td> ], start tag match failed [ <td> ] on line 155.

(tag-pair)